scripts/linter_utils_test.py

```diff
@@ -27,6 +27,19 @@
 class MemoizeTest(test_utils.GenericTestBase):
     """Tests for the memoize function."""
 
+    def test_memoize_always_returns_same_value(self):
+        lst = []
+        @linter_utils.memoize
+        def extend_then_return_lst():
+            """Appends 1 to the lst, then returns it."""
+            lst.append(1)
+            return lst
+
+        self.assertEqual(extend_then_return_lst(), [1])
+        self.assertEqual(lst, [1])
+        self.assertEqual(extend_then_return_lst(), [1])
```

Comment:
Add a comment here to explain what is happening (i.e. the actual function is not being called)